bundles/org.openhab.binding.proteusecometer/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="proteusecometer"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="EcoMeterS">
+		<label>Proteus EcoMeter S</label>
+		<description>Sensor for measuring water level of a cistern. Connected via USB</description>
+
+		<channels>
+			<channel id="temperature" typeId="Temperature"/>
+			<channel id="sensorLevel" typeId="SensorLevel"/>
+			<channel id="usableLevelInLitre" typeId="UsableLevelInLitre"/>
+			<channel id="usableLevelInPercent" typeId="UsableLevelInPercent"/>
+			<channel id="totalCapacity" typeId="TotalCapacity"/>
+		</channels>
+
+		<config-description>
+			<parameter name="usbPort" type="text" required="true">
+				<context>serial-port</context>
+				<label>USB Port</label>
+				<description>USB port the device is connected to i.e. /dev/ttyUSB0</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="Temperature">
+		<item-type>Number:CELSIUS</item-type>
```

Comment:
```suggestion
		<item-type>Number:Temperature</item-type>
```
See
https://www.openhab.org/docs/concepts/units-of-measurement.html#defining-channeltypes

if you try and create an item in OH3's main UI, this needs to match what is offered as an option. Please do a self review of the others as more than this one needs changing.